dgraph/Makefile

```diff
@@ -18,7 +18,14 @@ BIN            = dgraph
 BUILD         ?= $(shell git rev-parse --short HEAD)
 BUILD_DATE    ?= $(shell git log -1 --format=%ci)
 BUILD_BRANCH  ?= $(shell git rev-parse --abbrev-ref HEAD)
+
+# use a large version number v99 on master branch
+ifeq ($(BUILD_BRANCH),master)
+BUILD_VERSION ?= 'v20.11.0-'$(BUILD)
```

Comment:
Maybe add a note here that it needs to be bumped every few months.

Reply:
Done